MoltenVK/MoltenVK/API/vk_mvk_moltenvk.h

```diff
@@ -927,6 +928,19 @@ VKAPI_ATTR void VKAPI_CALL vkGetMTLTextureMVK(
     VkImage                                     image,
     id<MTLTexture>*                             pMTLTexture);
 
+/**
+* Returns, in the pMTLBuffer pointer, the MTLBuffer currently underlaying the VkImage.
```

Comment:
Please change several references in  copied comments from `VkImage` to `VkBuffer`.

Reply:
Thanks for reviewing. Correction applied.